chainer/functions/hierarchical_softmax.py

```diff
@@ -184,14 +184,16 @@ def create_huffman_tree(word_counts):
         raise ValueError('Empty vocabulary')
 
     q = six.moves.queue.PriorityQueue()
-    for w, c in six.iteritems(word_counts):
-        q.put((c, w))
+    # Use unique id when two entries have the same count.
```

Comment:
More detailed explanation is preferable.
* What do we use unique id for?
* What `unique id` indicate in this context?